cross/libaom/Makefile

```diff
@@ -24,6 +24,12 @@ PKG_WORK_DIR = $(WORK_DIR)/$(PKG_NAME)_$(PKG_VERS)-build
 CMAKE_USE_NASM = 1
 CMAKE_USE_NINJA = 1
 
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+# fix compilation with newer compilers
+CMAKE_ARGS += -DCMAKE_CXX_FLAGS="-D_GLIBCXX_USE_C99_MATH"
+endif
```

Comment:
This is not needed as already in the current code on master
Ref line 35+ https://github.com/SynoCommunity/spksrc/blob/master/cross/libaom/Makefile

Reply:
@th0ma7 thanks for the hint, this saves me the time of digging into this...